Mapper/BaseGroupedMapper.php

```diff
@@ -70,11 +70,17 @@ public function with($name, array $options = array())
             'collapsed' => false,
             'class' => false,
             'description' => false,
+            'label' => $this->admin->getLabelTranslatorStrategy()->getLabel($name, $this->getName(), 'group'),
             'translation_domain' => null,
             'name' => $name,
             'box_class' => 'box box-primary',
         );
 
+        // BC layer for manual label
+        if (strpos($name, '.')) {
+            $defaultOptions['label'] = $name;
+        }
```

Comment:
I think there is another black magic solution : use `func_get_args()`. Never occurred to me until now but that might be a good way to do BC hacks without modifying method signatures.